src/summary/cards/horizontal-label-value.css

```diff
@@ -0,0 +1,10 @@
+sup {
```

Comment:
CSS Modules are only local for the css selectors that are css classes. Your css here modifies *all `<sup>` elements that ever will exist in the app*. To fix, make sure you're only selecting `sup`'s within this component.

```suggestion
.root sup {
```